replaced most btns & removed styledComponent

FerdousFardin opened the pull request:
Change all Button elements to TouchableOpacity

srninnovations commented:
Still buttons:

PeopleSelector.jsx - Cancel and Confirm 
DeleteAllConfirm.jsx / Delete Confirmation - Delete All, Unlock, Close, Delete
OrderPlacedConfirmation - NEW ORDER and EDIT ORDER
OrderUpdatedConfirmation.jsx - NEW ORDER, PRINT KITCHEN RECEIPT, PRINT CUSTOMER RECEIPT, Close
SelectionConfirmation.jsx - Edit order, Clear table, Close
ViewModal.jsx - View button
Selection.js - (Delivery/Collection) Clear and Next

srninnovations reviewed (changes requested):
Still buttons:

PeopleSelector.jsx - Cancel and Confirm
DeleteAllConfirm.jsx / Delete Confirmation - Delete All, Unlock, Close, Delete
OrderPlacedConfirmation - NEW ORDER and EDIT ORDER
OrderUpdatedConfirmation.jsx - NEW ORDER, PRINT KITCHEN RECEIPT, PRINT CUSTOMER RECEIPT, Close
SelectionConfirmation.jsx - Edit order, Clear table, Close
ViewModal.jsx - View button
Selection.js - (Delivery/Collection) Clear and Next

FerdousFardin commented:
> Still buttons:
> 
> PeopleSelector.jsx - Cancel and Confirm DeleteAllConfirm.jsx / Delete Confirmation - Delete All, Unlock, Close, Delete OrderPlacedConfirmation - NEW ORDER and EDIT ORDER OrderUpdatedConfirmation.jsx - NEW ORDER, PRINT KITCHEN RECEIPT, PRINT CUSTOMER RECEIPT, Close SelectionConfirmation.jsx - Edit order, Clear table, Close ViewModal.jsx - View button Selection.js - (Delivery/Collection) Clear and Next

replaced everything now and also improved UX in the custom Item modal, fixed clear function for collection and delivery
